scripts/e2e.sh

```diff
@@ -25,7 +25,8 @@ echo "e2e_tower_username: '"$TOWER_USERNAME"'" >> e2e_test_config.yml
 echo "Bootstrapping tower in openshift project: ${TOWER_OPENSHIFT_PROJECT}"
 ansible-playbook -i ./inventories/hosts \
 playbooks/bootstrap.yml \
---extra-vars "@e2e_test_config.yml"
+--extra-vars "@e2e_test_config.yml" \
+-e cicd=true
```

Comment:
@StevenTobin this one can go in e2e_test_config.yml file.

Reply:
Makes sense, i'll move the vars to there